sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## 10.1.0-beta.1 (Unreleased)
 
+### Features Added
```

Comment:
Should we change the version here to 10.1.0 with a date, or is that handled by a separate process?

Reply:
I think in past PRs the date wasn't changed until it was approved by the SDK team, but I could be wrong about that. Regardless I do think the version should be changed

Reply:
Just checked and it does seem like that's the case re: the date